### DISCUSSION TITLE
Request For More Dracula Themed Wallpaper

### DISCUSSION THREAD

**Discussion by pspiagicw** [dracula/dracula-theme#538] [Show and tell] [answered]
# We Need More Wallpapers
We need more variety in the wallpapers.The existing wallpaper had become boring as I was using it for a year now.

So instead of downloading and using other wallpaper . I just recolored/rethemed the wallpaper I downloaded.I think this could be a welcome to the existing wallpaper collection.Keep in mind you might have seen these somewhere.I just changed all the colors to match dracula color pallete.I have designed about 25 for personal use.I think  you will have taste and the creativity to create better and more fitting wallpaper.

One thing to consider.I designed this for personal use so I did not care about the licensing , but showcasing on your website might bring licensing issue.
Hope you consider.Thank you in Advance.

Here are some examples I recolored.
![dracula005](https://user-images.githubusercontent.com/30765406/108583474-b0c94580-735f-11eb-99e8-2a372b71cbe4.jpg)
![dracula014](https://user-images.githubusercontent.com/30765406/108583480-b58df980-735f-11eb-8411-707443750713.png)
![dracula022](https://user-images.githubusercontent.com/30765406/108583486-bcb50780-735f-11eb-93dd-9726e2a74895.png)
![dracula001](https://user-images.githubusercontent.com/30765406/108583489-c0488e80-735f-11eb-91be-31520af5cd37.png)
![dracula002](https://user-images.githubusercontent.com/30765406/108583491-c2125200-735f-11eb-8a0c-af78617b16ef.png)
![dracula010](https://user-images.githubusercontent.com/30765406/108583494-c9396000-735f-11eb-8451-08d9dd72d624.png)

**Comment by bigpick** [comment, marked as answer]
Personally, I don't know that an official Dracula repo for wallpapers (other than the [existing one](https://github.com/dracula/wallpaper)) would be a good idea, since the scope is basically endless (and it'd be hard to keep the repo to a unifrom theme/style overtime). Also, wallpapers are pretty user-taste specific.  And as you've noted, it is really easy for anyone really to make their own. e.g, each is only two entities, and took a minute to make (these for 2K monitors landscape/portrait, 2560x1440):

<details>
  <summary>Click to expand!</summary>
  

![dracula_term_min_1_landscape](https://user-images.githubusercontent.com/9803299/111677589-bf3b3d80-87f5-11eb-971e-cf1c4d88ac9d.png)

![dracula_term_min_1_portrait](https://user-images.githubusercontent.com/9803299/111677588-bea2a700-87f5-11eb-9610-d5cff564b975.png)

---

![dracula_term_min_2_landscape](https://user-images.githubusercontent.com/9803299/111677582-bd717a00-87f5-11eb-81cd-0a12c54a9af2.png)

![dracula_term_min_2_portrait](https://user-images.githubusercontent.com/9803299/111677584-be0a1080-87f5-11eb-8ca8-2dcad01dc6ab.png)

</details>

(above done on Sketch, but should be possible on any design type sw that supports shapes+textbox+custom colors, Figma, Powerpoint, w/e)
